src/components/Inventory.js

```diff
@@ -1,58 +1,110 @@
+    fetchSeeds();
+  }, []);
+
+  const fetchSeeds = () => {
+    fetch('http://localhost:8080/inventory')
+      .then((response) => {
+        // return response.text();
+        return response.json();
+      })
+      .then((json) => {
+        setSeeds(json);
+        // setSeeds(text);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  };
+
+
+  //table creation
+  const columns = [{
+    dataField: 'seedName',
+    text: 'SEED NAME', 
+    sort: true,
+    editable: false
+  }, {
+    dataField: 'qty',
+    text: 'WEIGHT(g)',
+    sort: true
+  }];
+
+  const detailColumns = [{
+    dataField: 'blackoutTime',
+    text: 'Blackout Time'
+  }, {
+    dataField: 'harvestTime',
+    text: 'Harvest Time'
+  }, {
+    dataField: 'seedPresoak',
+    text: 'Presoak'
+  }, {
+    dataField: 'germinationTime',
+    text: 'Germination Time'
+  }, {
+    dataField: 'lot',
+    text: 'Lot'
+  }];
+
+  const expandRow = {
+    renderer: row => (
+      <BootstrapTable 
+        keyField='id'
+        data = { seeds }
+        columns = { detailColumns }
+      />
+    )
+  }
+
   return (
     <div>
       <div class="inventory">
         <h2 class="item">INVENTORY</h2>
-        <div class="item">
-          <h4>SORT BY</h4>
-          <Form.Select >
-            <option>Lot No.</option>
-            <option>Seed Name</option>
-            <option>Weight</option>
-            <option>Status</option>
-          </Form.Select>
-        </div>
         <Button variant="dark" class="item" id="addSeedsBtn">ADD SEEDS</Button>
       </div>
-      <Table striped bordered hover>
+
+      <BootstrapTable 
+        keyField='id'
+        data = { seeds }
+        columns = { columns }
+        expandRow = { expandRow }
+        cellEdit= { cellEditFactory({mode: 'dbclick' })}
+      />
+
+
+{/* commented out old table mapping  */}
+      {/* <Table striped bordered hover>
         <thead>
           <tr>
-            <th>LOT NO.</th>
             <th>SEED NAME</th>
-            <th>COUNT</th>
             <th>WEIGHT(g)</th>
-            <th>STATUS</th>
           </tr>
         </thead>
-        <tbody>
-          <tr>
-            <td>1</td>
-            <td>Dandelion</td>
-            <td>200</td>
-            <td>3</td>
-            <td>Available</td>
-          </tr>
-          <tr>
-            <td>2</td>
-            <td>Amaranth</td>
-            <td>300</td>
-            <td>5</td>
-            <td>Available</td>
-          </tr>
-          <tr>
-            <td>3</td>
-            <td>Basil Genovese</td>
-            <td>400</td>
-            <td>1.5</td>
-            <td>Planted</td>
-          </tr>
-        </tbody>
-      </Table>
+          <tbody>
+          {seeds.map((item, i) => (
+                    <tr key={i}>
+                        <td>{item.seedName}</td>
+                        <td>{item.qty}</td>
+                    </tr>
+          ))}
+          </tbody>
+      </Table> */}
```

Comment:
Code looks great. 
As a general rule of thumb, remove commented out code.
I haven't run this locally but looks good reading through it.